_posts/2023-03-02-dafny-4-released.markdown

```diff
@@ -17,10 +17,9 @@ For additional information on Dafny, please see:
 * [Dafny on GitHub](https://github.com/dafny-lang/dafny)
 
 ## Dafny 4.0 Release Highlights
-
 If you haven’t had a look at Dafny since 3.0, you’re missing out on a ton of exciting new features and enhancements! Here are some of the recent highlights since Dafny 3.0. Some of these features have been available in prior releases, but are now enabled by default in 4.0.
```

Comment:
This first stmt reads a bit strange to me (as some of these were not on by default). How about phrase it as "We have added a lot of exciting new features and enhancements to the Dafny language and ecosystem since Dafny 3.0. Here are some of the recent highlights."